### ISSUE TITLE
Incompatibility with Advanced Telematics TUF implementation

### ISSUE THREAD

**Issue by doanac** [theupdateframework/notary#1414]
This could easily be a "won't fix", but I thought I'd mention it. As reported [here](https://github.com/advancedtelematic/aktualizr/issues/1073), the aktualizr program is rejecting the timestamp.json served by Notary. The issue is that Notary serves the JSON as `json["signed"]["meta"]["snapshot"]`, but aktualizr assumes the key is "snapshot.json".

According to [this comment](https://github.com/advancedtelematic/aktualizr/issues/1073#issuecomment-458515997), this could be a bug in Notary.

Is this fixable in Notary w/o causing compatibility issues?

**Comment by justincormack** [comment]
Yes, I noticed some issues with the filenames recently, and my initial conclusions were that (a) Notary is incorrect in some cases and (b) the spec also may need some changes. I was going to spend some time on this soon.

I think that we will be able to fix Notary in a backwards compatible way by serving on multiple paths as they cannot conflict.
